cypress/integration/models/developer/applicationinventory/analysis.ts

```diff
@@ -228,18 +228,18 @@ export class Analysis extends Application {
             .closest(trTag)
             .within(() => {
                 cy.get(analysisColumn)
-                    .find("div > div")
+                    .find("div > div:nth-child(2)", { timeout: 10800000 }) // 3h
+                    .should("not.have.text", "Not started")
+                    .and("not.have.text", "Scheduled")
+                    .and("not.have.text", "In-progress")
                     .then(($a) => {
                         if ($a.text().toString() != status) {
                             // If analysis failed and is not expected then test fails.
                             if ($a.text().toString() == "Failed" && status != "Failed") {
                                 expect($a.text().toString()).to.eq("Completed");
```

Comment:
toString() will again create new object, If it is pooled frequently it will affect memory.
Is there is another way to see if it is completed? - Like CLI way? checking some folder for some file?

Reply:
I will take a look into it, anyways after the refactor, this method will be called only once per test so this won't be the reason why tests are crashing.